src/any_agent/tools/a2a.py

```diff
@@ -104,6 +105,7 @@ async def _send_query(query: str) -> str:
 
         Args:
             query (str): The query to perform.
+            task_id (str): The task id to use for the conversation. Defaults to a new uuid. If you want to continue the conversation, you should provide the same task id that you received in a previous response.
```

Comment:
This is the attempt to give the agent using the tool an idea of how to use the tool, so that you don't need to specifically prompt engineer the system/user prompt when creating the agent, all the info it needs should be here.